pkg/controller/direct/bigquery/table_mapper.go

```diff
@@ -631,7 +695,11 @@ func BigQueryTableSpec_ToProto(mapCtx *direct.MapContext, in *krmv1beta1.BigQuer
 	out.FriendlyName = direct.ValueOf(in.FriendlyName)
 	out.Description = direct.ValueOf(in.Description)
 	out.Labels = in.Labels
-	out.Schema = Table_Schema_ToProto(mapCtx, in.Schema)
+	if out.Labels == nil {
+		out.Labels = map[string]string{}
+	}
+	out.Labels["managed-by-cnrm"] = "true"
```

Comment:
Nit: I suggest this to be done in the controller instead of mapper. I think this is a controller level customization, not mapper level.